README.md

```diff
@@ -135,6 +135,31 @@ var container = new Blister();
 container.register(provider);
 ```
 
+#### Creating new contexts
+
+Some scenarios may require different instances of a DI container. For example, we may want to have different DI containers to handle different requests of a server (having specific dependencies for the current request).
+
+In order to be able to define global dependencies and context-specific dependencies, we can create a new context from a container, which is just a container that inherits dependencies from the original one.
+
+Example:
+
+```javascript
+var container = new Blister();
+container.service('logger', function() {
+  return 'system logger';
+});
+
+var context = container.createContext();
+context.get('logger'); // 'system logger'
+
+context.service('logger', function() {
+  return 'request logger';
+});
+context.get('logger'); // 'request logger'
+```
+
+__NOTE__: To prevent errors with services, which are cached, they are bound to the dependencies of the context/container where they are defined. That way, accessing a service through a context will not cache it in the container with a dependency of a sub-context.
```

Comment:
You have to read this at least 3 times to understand...
"To prevent errors", what type of errors?
Why as a user of your lib, should I know this? Does it has practical impact on how to use your lib?
Maybe an example instead of this "cryptic" sentence?